ClosedXML/Excel/SaveOptions.cs

```diff
@@ -17,5 +17,7 @@ public SaveOptions()
         public Boolean EvaluateFormulasBeforeSaving { get; set; } = false;
         public Boolean GenerateCalculationChain { get; set; } = true;
         public Boolean ValidatePackage { get; set; }
+        public Boolean ConsolidateConditionalFormatRanges { get; set; } = true;
+        public Boolean ConsolidateDataValidationRanges { get; set; } = true;
```

Comment:
I believe the properties are ordered alphabetically. Put these two accordingly.

Reply:
codemaid applied in new version